Simple solvers/JV.py

```diff
@@ -51,30 +51,30 @@
     root1new = round(m + k,2) 
     root2new = round(m - k,2)
 
+    print('The roots for the equation using the new method are '+ str(root1new) + ' and ' + str(root2new))
+   
     #verifying both methods produce the same results 
     if (root1==root1new and root2==root2new) or (root1==root2new and root2==root1new): #sometimes the roots are the same but in a different order
-        print('The roots for the equation using the new method are '+ str(root1new) + ' and ' + str(root2new))
         print('Both methods work!')
 
     else:
-        print('The roots for the equation using the new method are '+ str(root1new) + ' and ' + str(root2new))
         print('The second method does not work :(')
         
 elif z == 0:
     rootnew = round(m,2)
     print('The distance from the roots is 0 because there is only one root.')
-   
+    
+    print('The root for the equation using the new method is ' + str(rootnew))
+
     #verifying both methods produce the same results
     if root==rootnew:
-        print('The root for the equation using the new method is ' + str(rootnew))
         print('Both methods work!')
```

Comment:
Could you also refactor this message?  It is duplicated more than once which makes it hard to maintain.